src/vehicle.cpp

```diff
@@ -1103,23 +1103,23 @@ bool vehicle::start_engine( const int e )
     }
 
     if( eng.faults().count( fault_immobiliser ) ) {
-        add_msg( _( "The %s cannot start with a faulty immobiliser." ), eng.name().c_str() );
+        add_msg( _( "The %s makes a long beeping sound." ), eng.name().c_str() );
         return false;
     }
 
     if( eng.faults().count( fault_starter ) ) {
-        add_msg( _( "The %s cannot start with a faulty starter motor." ), eng.name().c_str() );
+        add_msg( _( "The %s makes a single clicking sound." ), eng.name().c_str() );
         return false;
     }
 
     int joules = eng.base.engine_start_energy( g->temperature );
     if( fuel_left( fuel_type_battery, true, true ) < joules ) {
-        add_msg( _( "The %s need at least %i battery charges to start" ), eng.name().c_str(), joules );
+        add_msg( _( "The %s makes a rapid clicking sound." ), eng.name().c_str(), joules );
```

Comment:
1. Diesel engines don't make rapid clicks when too cold to start
2. Hiding information from the player (required charges) is bad UI design
3. `printf` format error

The first two problems are repeated elsewhere in this PR and the second is probably magnified by `i18n` (a lot can be lost in translation).